### DISCUSSION TITLE
Reach out to Tom Carruthers' MSE tool team

### DISCUSSION THREAD

**Discussion by ChristineStawitz-NOAA** [NOAA-FIMS/collaborative_workflow#127] [General]

**Comment by ChristineStawitz-NOAA** [comment]
Does anyone have an interest in keeping tabs on this group? https://github.com/Blue-Matter/openMSE

**Comment by k-doering-NOAA** [comment]
Possible @nathanvaughan-NOAA  or me? (Not to volunteer you, Nathan) We had a call with this group maybe like a year ago to discuss MSE tools (SSMSE and openMSE).

**Comment by nathanvaughan-NOAA** [comment]
Sounds good to me @k-doering-NOAA , either of us can do it. @ChristineStawitz-NOAA what is the intended outcome for this? Do we want to invite Tom et al to present on their openMSE work, update them on what is happening with FIMS, invite them to one of our meetings, just keep an eye on the openMSE package, or something else? Just wanted to make sure we are on the same page.

**Comment by ChristineStawitz-NOAA** [comment]
Thanks, I was creating a bunch of these in a hurry and should have specified more details.

I am thinking more about understanding the approach used in openMSE and thinking about how it could apply to FIMS development, for example, when we are making decisions like should we use S3 or S4 classes, does one of them allow us to maintain compatibility with open MSE? I would also like to let them know that we exist somehow - though we are light on details right now, you could share the Github site so they can track if we have a major release. To be honest, I don't know enough about openMSE to even know exactly the right approach - with CASAL2 we have had a couple of meetings with a subset of the FIMS group - but I am trying to avoid reinventing the wheel here across the different packages/new packages.

**Comment by k-doering-NOAA** [comment]
@ChristineStawitz-NOAA , I like the idea of just having a meeting with a subset of the FIMS group. I can propose this to the openMSE folks, if you think now would be a good time to do this!

**Comment by nathanvaughan-NOAA** [comment]
Thanks for jumping in Kathryn. Based on our previous meeting with Tom's group and my understanding of openMSE I don't think there is any real overlap with FIMS yet? Integration of FIMS with openMSE is just about extracting the population/fishery dynamics from FIMS and putting them into the openMSE OM so I don't believe our S3/S4 class choice will have an impact. I think we may get more utility out of a meeting once we have a base model running and are considering approaches for incorporating MSE/forecasting/simulation capabilities. Useful interface decisions, simulation choices, and diagnostic outputs are things that openMSE/DLMtool have had lots of time to perfect. Not that I have an issue with setting up a meeting now if that is what the consensus decides. Just don't want to take up people's time getting ahead of ourselves. I think the CASAL/GADGET/FLR/WHAM/etc discussions are much more valuable at this stage of development for guiding the initial assessment model development.

**Comment by k-doering-NOAA** [comment]
Good points, Nathan.

BTW openMSE uses mostly an S4 object system, which I believe is true of FLR, from what I remember. But I think Nathan's right that our design choices for FIMS need not be limited.
